pkgs/development/python-modules/gin-config/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, enum34
+, absl-py
+, tensorflow
+}:
+
+buildPythonPackage rec {
+  pname = "gin-config";
+  version = "0.1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zqzhg8nqdqjx0sjrsrs9bpyiaqqlyihqhhy2ijrpp62x9rjllga";
+
+  };
+
+  buildInputs = [ six enum34 tensorflow ];
```

Comment:
I'd add an optional parameter `withTensorflow ? false` here as it seems to be optional according to `setup.py`: https://github.com/google/gin-config/blob/master/setup.py#L51

This might look like this then:

``` nix
{
  # ...
  propagatedBuildInputs = [ six enum34 ] ++ lib.optional withTensorflow tensorflow;
}
```

From my understanding this project doesn't need to be used for ML projects and in such a case I doubt that you want to have `tensorflow` in your build closure :)

Reply:
I wrote a separate comment already, but no, there should not be an option for it.